addons/account/models/account_account.py

```diff
@@ -506,12 +506,19 @@ def default_get(self, default_fields):
         return super(AccountAccount, contextual_self).default_get(default_fields)
 
     @api.model
-    def _order_by_frequency_per_partner(self, company_id, partner_id, move_type=None, limit=None):
+    def _get_most_frequent_accounts_for_partner(self, company_id, partner_id, move_type, filter_never_used=False, limit=None):
```

Comment:
Couldn't we find a better name for the parameter `filter_never_used`? Something like `filter_never_user_accounts`, `filter_accounts` or `filter_accounts_out`?